extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1590,6 +1601,20 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperationWithProgress<QuickStartProjectResult, QuickStartProjectProgress> GenerateAsync();
     }
 
+    // Extensions implement this and is provided to Dev Home when CreateChatStyleGenerationOperation is called.
+    // This allows to have an async operation can be started but also additional context reported back to Dev Home
+    // as progress is being made such as a detailed log out output through the adaptive card session.
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
```

Comment:
Do we need to increment contract version throughout.

Reply:
Sarah stated in a previous thread: If the contract version is being incremented here, we may also need to change the VersionOfSDK value inside build/azure-pipelines.yml.

Reply:
I believe I saw the current contract version is 7, so I updated all of the new ones I'm adding in to be contract version 8.

Reply:
In addition to the aforementioned change, I also incremented the VersionOfSDK value in the azure-pipeliens.yml file.